docs/usecases/running_tests_on_cluster_not_deployed_by_ocsci.rst

```diff
@@ -0,0 +1,195 @@
+How to execute tests on a cluster not deployed by ocs-ci?
+=========================================================
+
+Running ocs-ci test cases on clusters not deployed by ocs-ci itself is
+possible, but because ocs-ci was not designed with this use case in
+mind, it requires few additional steps as explained below. Without
```

Comment:
s/ but because ocs-ci was not designed with this use case in mind// , it was in the early design phase to run on existing cluster

Reply:
I could rephrase that in a sense that this was outside of main focus so far.

Reply:
addressing via a04b0892cd314eaece374b6304cb83fced019564